src/api/index.ts

```diff
@@ -6,49 +6,33 @@
  */
 
 import axios from 'axios'
+import {stringify} from 'query-string'
 
-let api: any
 let baseUrl: string = process.env.API_ENDPOINT
 	? process.env.API_ENDPOINT
 	: 'https://jsonplaceholder.typicode.com'
 
-const getData = res => res.data
+export const api: any = axios.create({baseURL: baseUrl})
 
-const joinUrl = (endpoint, params?, query?): string => {
-	let joinedUrl = [`${baseUrl}/${endpoint}`]
+const getData = res => res.data
+const joinUrl = (path, params?, query?): string => {
+	let joinedUrl = `${path}`
 	if (params) {
```

Comment:
Should check if something exists instead of its truthy value.
`params?: number,` is declared in modelAction.ts so technically 0 is a possible case.
`joinUrl('/somepath', 0, 'somequery')` should return `/somepath/0?somequery` instead of `/somepath?somequery` 
This is just a corner case. For now it looks good for me